Makefile

```diff
@@ -101,4 +104,5 @@ check: checkcommits log-parser
 	log-parser \
 	openshift \
 	swarm \
+	security \
```

Comment:
Surprised @jodh-intel did not spot this is not actually alphabetical ;-)

Reply:
I blame lack of :coffee: :smile: